config/initializers/sidekiq.rb

```diff
@@ -3,6 +3,7 @@
     config.redis = {
       url: ENV.fetch("REDIS_WORKER_URL"),
     }
+    config.logger.level = Logger::WARN
```

Comment:
Do we want to lower the level of all sidekiq jobs? The big volume one is the BigQuery one and you've already handled that.

Reply:
This only removes the "start" and "stop" default sidekiq logs. Since we use activejob, we also get the "Processing" and "Processed" logs.